app/assets/styles/styles.css

```diff
@@ -172,24 +172,36 @@ a {
 }
 
 .donate {
-  height: 300px;
-  font-size: 1.5rem;
-  margin-left: 2em;
-  margin-right: 2em;
+  padding: 1em 3em;
+  margin-left: auto;
+  margin-right: auto;
   background-image: linear-gradient(rgba(21, 21, 78, 0.5), rgba(21, 21, 78, 0.6)), url("/images/newsletter/salvia-team.webp");
   background-size: cover;
   background-position: center center;
   background-repeat: no-repeat;
   border: 5px solid #096a97;
   color: #fff;
   text-align: center;
+
 }
 
 .donate__btn {
-  font-size: 1.5rem;
-  font-weight: bold;
-  text-decoration: none;
   color: white;
   background-color: #6fcf97;
-  padding: 1em 1.5em;
+  padding: 0.5em 1em;
+  display: block;
+  width:3em;
+  font-size: 1.5rem;
+  margin-left: auto;
+  margin-right: auto;
+}
+.donate__paragraph {
+  width:25em;
+  font-size: 1.5rem;
+  margin-bottom: 1.5em;
+  margin-left: auto;
+  margin-right: auto;
+}
+.donate__h2 {
```

Comment:
Getting there! 

1. We always have a space between each ruleset in our code base. 
1. We prefer to use semantic class names rather than use the html element. 

There are many examples of how we are naming classes in the code base. Take a look at the bulletin code or some of the PRs #127

Reply:
Resolved pairing